docs/02-app/02-api-reference/05-next-config-js/turbo.mdx

```diff
@@ -8,9 +8,19 @@ description: Configure Next.js with Turbopack-specific options
 
 > **Warning**: These features are experimental and will only work with `next --turbo`.
 
+Turbopack can be customized to transform different files and change how modules are resolved.
+
+> **Good to know**:
+>
+> - Turbopack for Next.js does not require loaders nor loader configuration for built-in functionality. Turbopack has built-in support for css and compiling modern JavaScript, so there's no need for `css-loader`, `postcss-loader`, or `babel-loader` if you're using `@babel/preset-env`.
+
```

Comment:
We could probably combine **warning**, and **good to know**. wdyt?

```suggestion
Turbopack can be customized to transform different files and change how modules are resolved.

> **Good to know**:
>
> - These features are experimental and will only work with `next --turbo`.
> - Turbopack for Next.js does not require loaders nor loader configuration for built-in functionality. Turbopack has built-in support for css and compiling modern JavaScript, so there's no need for `css-loader`, `postcss-loader`, or `babel-loader` if you're using `@babel/preset-env`.

```

Reply:
💯